### PR TITLE
[KaNN] Add support to open, decode and visualize a KaNN graph

### PR DESCRIPTION
**Pull request by zephfj**
…NN graph

### CODE REVIEW

File: source/kann-metadata.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        "category": "Data"
+    },
+    {
+        "name": "MaxWithConstant",
+        "category": "Transform"
+    },
+    {
+        "name": "FullMapping",
+        "category": "Transform"
+    },
```
Comment (LucasMahieu) [inline source/kann-metadata.json]: this is an abstract class, it will never be serialized in a model

File: source/kann-metadata.json [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        "name": "Exp",
+        "category": "Activation"
+    },
+    {
+        "name": "Mish",
+        "category": "Activation"
+    },
+    {
+        "name": "WindowedMapping",
+        "category": "Transform"
```
Comment (LucasMahieu) [inline source/kann-metadata.json]: this is an abstract class, it will never be serialized in a model

File: source/kann-metadata.json [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    },
+    {
+        "name": "GeLU",
+        "category": "Activation"
+    },
+    {
+        "name": "Layer",
+        "category": "Layer"
+    },
```
Comment (LucasMahieu) [inline source/kann-metadata.json]: this is an abstract class, it will never be serialized in a model

File: source/kann-metadata.json [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+        "name": "DepthwiseConvolution",
+        "category": "Layer"
+    },
+    {
+        "name": "Softmax",
+        "category": "Activation"
+    },
+    {
+        "name": "SimpleMapping",
+        "category": "Transform"
```
Comment (LucasMahieu) [inline source/kann-metadata.json]: this is an abstract class, it will never be serialized in a model

File: source/kann-metadata.json [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+        "name": "SiLU",
+        "category": "Activation"
+    },
+    {
+        "name": "ImageSMEM",
+        "category": "Tensor"
+    },
+    {
+        "name": "Image",
+        "category": "Tensor"
```
Comment (LucasMahieu) [inline source/kann-metadata.json]: this is an abstract class, it will never be serialized in a model

File: source/kann-metadata.json (hunk 1 above)
Comment (zephfj) [reply source/kann-metadata.json]: I added a filter when generating kann-metadata.json to not consider abstract classes

**Pull request merged**